intelmq/bots/collectors/mail/collector_mail_attach.py

```diff
@@ -22,7 +22,9 @@ def process(self):
                               self.parameters.mail_user,
                               self.parameters.mail_password,
                               self.parameters.mail_ssl)
-        emails = mailbox.messages(folder=self.parameters.folder, unread=True)
+        emails = mailbox.messages(folder=self.parameters.folder, unread=True,
+                                  sent_to=getattr(self.parameters, "sent_to", None),
```

Comment:
The [default values used in imbox](https://github.com/martinrusev/imbox/blob/master/imbox/query.py#L20) are `False` not `one`None`, but I guess the result is the same?